nameko/amqp.py

```diff
@@ -43,7 +43,7 @@ class TestTransport(Transport):
 
 def verify_amqp_uri(amqp_uri):
     connection = Connection(amqp_uri)
-    if connection.transport_cls != 'amqp':
+    if connection.transport_cls not in ('amqp', 'pyamqp'):
```

Comment:
sorry, read this the wrong way around. you are _adding_ another transport which _can_ use the `TestTransport`